app/src/main/java/org/p2p/wallet/push_notifications/repository/NotificationServiceRepository.kt

```diff
@@ -0,0 +1,8 @@
+package org.p2p.wallet.push_notifications.repository
+
+import org.p2p.wallet.push_notifications.model.DeviceToken
+
+interface NotificationServiceRepository {
+    suspend fun sendDeviceToken(params: Map<String, Any>): DeviceToken
```

Comment:
NotificationServiceRepository  -> DeviceTokenRepository then?

Reply:
It could be